asset/js/widget/BaseInput.js

```diff
@@ -49,18 +49,23 @@ define(["../notjQuery", "Completer"], function ($, Completer) {
             $(this.input).on('input', this.onInput, this);
             $(this.input).on('keydown', this.onKeyDown, this);
             $(this.input).on('keyup', this.onKeyUp, this);
-            $(this.input).on('blur', this.onInputBlur, this);
             $(this.input).on('focusin', this.onTermFocus, this);
             $(this.termContainer).on('input', '[data-label]', this.onInput, this);
             $(this.termContainer).on('keydown', '[data-label]', this.onKeyDown, this);
-            $(this.termContainer).on('keyup', '[data-label]', this.onKeyUp, this);
-            $(this.termContainer).on('focusout', '[data-index]', this.onTermFocusOut, this);
-            $(this.termContainer).on('focusin', '[data-index]', this.onTermFocus, this);
+
+            if (! this.isReadOnlyMode()) {
+                $(this.termContainer).on('keyup', '[data-label]', this.onKeyUp, this);
+                $(this.input).on('blur', this.onInputBlur, this);
+                $(this.termContainer).on('focusout', '[data-index]', this.onTermFocusOut, this);
+                $(this.termContainer).on('focusin', '[data-index]', this.onTermFocus, this);
+            }
 
             // Copy/Paste
-            $(this.input).on('paste', this.onPaste, this);
-            $(this.input).on('copy', this.onCopyAndCut, this);
-            $(this.input).on('cut', this.onCopyAndCut, this);
+            if (! this.isTermDirectionVertical()) {
```

Comment:
I've changed my mind here. It makes no real sense for the use-case in https://github.com/Icinga/icingaweb2/pull/4691, but this is a library where this is irrelevant. Just because the direction isn't horizontal shouldn't be a reason for the copy paste functionality to not work.